docs/community/contributing.md

```diff
@@ -79,6 +79,9 @@ When you submit a pull request, the DCO-bot will automatically assess whether yo
 
 For more details and the exact DCO text, you can visit [Developer Certificate of Origin](developercertificate.org).
 
+## Adopters
+If you are leveraging Akri for your solution, we highly encourage you to add your organization or project information to our [adopter list](https://github.com/project-akri/akri/blob/main/ADOPTERS.md). Please submit a pull request against the [ADOPTERS.md in the Akri GitHub](https://github.com/project-akri/akri/blob/main/ADOPTERS.md). Your feedback is highly appreciated and would help us to justify the future investment in this project.
```

Comment:
As worded, it seems to suggest that we are hesitant to invest in the project. Can we slightly modify it to:
```suggestion
If you are leveraging Akri for your solution, we highly encourage you to add your organization or project information to our [adopter list](https://github.com/project-akri/akri/blob/main/ADOPTERS.md). Please submit a pull request against the [ADOPTERS.md in the Akri GitHub](https://github.com/project-akri/akri/blob/main/ADOPTERS.md). Your participation helps us prioritize feature development in the project.
```